File: Java/20-08-24/src/com/lop/project/Employee.java
package com.lop.project;

class Employee {
    String name;
    String employeeId;
    String position;
    double salary;

    Employee(String name, String employeeId, String position, double salary) {
        this.name = name;
        this.employeeId = employeeId;
        this.position = position;
        this.salary = salary;
    }

    double calculateAnnualSalary() {
        return salary * 12;
    }
}

File: C#/Homework 9/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Homework_9
{
    public class Employee : EmployeeBase
    {
        public Employee(string employeeId, string name, string position, DateTime startDate, decimal salary)
            : base(employeeId, name, position, startDate, salary)
        {
        }
    }

}
